content_api/items/service.py

```diff
@@ -253,6 +253,8 @@ def _process_item_associations(self, item):
         allowed_items = {}
         if item.get('associations'):
             for _k, v in item.get('associations', {}).items():
+                if v is None:
```

Comment:
maybe `if not v:` would be better, it would also handle `{}` or `False` besides `None`